.github/workflows/ci_cron_weekly.yml

```diff
@@ -93,7 +93,7 @@ jobs:
     # of using system libraries and using pytest directly.
 
     runs-on: ubuntu-20.04
```

Comment:
I don't think ubuntu-20.04 supports py3.12.
@pllim can we change this to ubuntu-latest?

Reply:
Or Ubuntu-22.04?

Reply:
I tried bumping

Reply:
Hm. That's not the reason. Something about Docker.

Reply:
I can reduce it back to ubuntu-20.04, unless we should just keep it on ubuntu-latest?

Reply:
Latest keeps it properly updated. SGTM.